README.md

```diff
@@ -116,6 +116,7 @@ GoLang: SDK|Every Thursday alternating between 10:00 and 15:00 PDT|[Google Doc](
 Instrumentation: General|Every Tuesday at 16:00 PT|[Google Doc](https://docs.google.com/document/d/1dWHhyXnfVife-cQ2DW5-d5Ldp1Lq8Rre2UsHpyo8cEE/)|[Zoom](https://zoom.us/j/6729396170?pwd=UFRIL0U0VzczUk1RK0hYaXJITXVldz09)|[Slack](https://cloud-native.slack.com/archives/C01QZFGMLQ7)
 Instrumentation: Messaging|Every Thursday at 8:00 PT|[Google Doc](https://docs.google.com/document/d/1dWHhyXnfVife-cQ2DW5-d5Ldp1Lq8Rre2UsHpyo8cEE/)|[Zoom](https://zoom.us/j/8203130519?pwd=elFSMWxOSVN0Qjk3cVJ1dzZoSnpnZz09)|[Slack](https://cloud-native.slack.com/archives/C01QZFGMLQ7)
 Java: SDK + Instrumentation|Every Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1WK9h4p55p8ZjPkxO75-ApI9m0wfea6ENZmMoFRvXSCw/)|[Zoom](https://zoom.us/j/8203130519)|[SDK](https://cloud-native.slack.com/archives/C014L2KCTE3) and [Instrumentation](https://cloud-native.slack.com/archives/C0150QF88FL) Slack channels|
+Java: JVM Metrics|Alternating: Monday at 10:00 PT, Wednesday at 10:00 *CET*|[Google Doc](https://docs.google.com/document/d/1WK9h4p55p8ZjPkxO75-ApI9m0wfea6ENZmMoFRvXSCw/)|[Zoom](https://zoom.us/j/8203130519)|[SDK](https://cloud-native.slack.com/archives/C014L2KCTE3) and [Instrumentation](https://cloud-native.slack.com/archives/C0150QF88FL) Slack channels|
```

Comment:
Is this using the cloud-native zoom channel?   We've been putting the password in the URL (e.g. see the Java SiG one above).

QQ: Did you check to make sure this doesn't conflict with other Zoom's at the same time?   If not I can go verify (we only have a limited number of zoom rooms).

Reply:
```suggestion
Java: JVM Metrics|Alternating: Monday at 10:00 PT, Wednesday at 10:00 *CET*|[Google Doc](https://docs.google.com/document/d/1WK9h4p55p8ZjPkxO75-ApI9m0wfea6ENZmMoFRvXSCw/)|[SDK](https://cloud-native.slack.com/archives/C014L2KCTE3) and [Instrumentation](https://cloud-native.slack.com/archives/C0150QF88FL) Slack channels|
```

We just merged in a change that removes the hardcoded Zoom links, as the calendar is now the source of truth. Removed this link so that your PR works in the new table.